### DISCUSSION TITLE
stackviewの制約について

### DISCUSSION THREAD

**Discussion by atsujr** [lit-iphone-course/lit-iphone-course#78] [Question-original] [answered]
(1) 実現したいこと
tableviewのセルのデザインを、以下の画像のようにしたい
(2) 実現するために自分が試した内容とその結果
前回の質問で教わったことを参考に、stackviewに色々な制約をつけ、さらにtableviewcellに影を追加した。が、画像の大きさが自分が実装したい大きさにならない。(自分が思ってるサイズよりも小さい)
(3) 調べた内容や資料
前回のstackvireに関するGithub discussion
(4) 原因だと思われそうな箇所(ソースコードをコピペして貼る)
みたところ、stackview周りの制約は間違えていないため、なぜ画像の大きさが変わらないかがわからないです。
以下にリンクを添付しておきます。ご確認、よろしくお願いします。
https://github.com/atsujr/BookList
![スクリーンショット 2022-09-28 15 21 42](https://user-images.githubusercontent.com/105122292/193002422-8ce235ef-c13e-4578-abbe-f9eebedc4864.png)

![IMG_8566](https://user-images.githubusercontent.com/105122292/193002572-8a1da61a-6242-40ba-8254-450b7ace8919.PNG)

**Comment by txxxxc** [comment, marked as answer]
多分なんだけど画像に付けてる比率の制約(140:81)と実際に取る写真の比率(16:9)が違うことでレイアウトが崩れてるっぽいので、一旦画像に付けてる制約を16:9にしてやってみてもらえるとレイアウトの調整がやりやすくなる思いますー！

**Reply by atsujr** [reply]
16:9にサイズを変更して、やってみたんですけどさっきと結果が一緒でした、、、
下の画像から確認してみたところ、stackviewの設定が間違えているのかUIimageviewの全体に画像が表示されていませんでした。
![スクリーンショット 2022-09-29 19 56 26](https://user-images.githubusercontent.com/105122292/193013733-673ceed5-652e-4286-ac6a-10152774564c.png)

**Reply by txxxxc** [reply]
あーそれUIImageViewの設定やな
これ見ながらやってみて欲しい↓

https://qiita.com/Saayaman/items/a23519ff5a8ad287cf20

**Reply by atsujr** [reply]
完全に忘れてました、、、
できました！本当にありがとうございます！
